README.md

```diff
@@ -146,6 +146,14 @@ with app_context():
 
 ![image](resources/screenshot-nD-image.png)
 
+You can draw lines and polygons on an image, including selection and adjustment of shapes and vertices, and control over fill and stroke color. Run `examples/add_shapes.py` to generate and interact with the following example.
+
+![image](resources/screenshot-add-shapes.png)
+
+You can also paint pixel-wise labels, useful for creating masks for segmentation, and fill in closed regions using the paint bucket. Run `examples/labels-0-2d.py` to generate and interact with the following example.
+
+![image](resources/screenshot-add-labels.png)
+
 ## plans
 
 We're working on several features, including 
```

Comment:
Here can you remove "shape-based annotation (for drawing polygons and bounding boxes)" and 
"region labeling (for defining segmentation)" as we already have these features in the examples above. Could add as a second bullet "Support for multiple canvases" or something else, I'm open.

Reply:
fixed!